Portfolios/Brutalist/style.css

```diff
@@ -7,107 +6,118 @@ FLEXBOX
 */
 
 body {
-  margin-left: 60px;
-  margin-top: 20px;
+	margin-left: 60px;
```

Comment:
Hey there,

Thanks for your contribution :)

I think what has happened here is that your code formatter has changed the tab rules of the file. If you could recommit with the repo defaults, the changes displayed will be limited to just the important bits! 

Many thanks :)

Reply:
Okay, I will do that.

Reply:
Hey!
Okay so I am totally new to the PR thing but I tried to make some changes and i don't know what to do from here. I have made some more commits, could you please check and tell me if Tabs space is still a problem? I don't know how to do what you asked. I am sorry